packages/x-components/src/components/panels/base-tabs-panel.vue

```diff
@@ -0,0 +1,332 @@
+<template>
```

Comment:
Just a question, should we add the `tablist` role to the list component, and the `tab` role to the `button`?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

Reply:
Good point! I've added the `tablist`, `tab`, and also `tabpanel` roles.